src/ui/components/Nav.tsx

```diff
@@ -16,7 +16,8 @@ export async function Nav() {
 	const checkoutId = cookies().get("checkoutId")?.value || "";
 
 	const checkout = await Checkout.find(checkoutId);
-	const lines = checkout ? checkout.lines : [];
+
+	const lineCount: number = checkout ? checkout.lines.reduce((result, line) => result + line.quantity, 0) : 0;
```

Comment:
```suggestion
	const lineCount = checkout ? checkout.lines.reduce((result, line) => result + line.quantity, 0) : 0;
```

Using _type inference_ is more idiomatic.